docs/messages_guide/src/asciidoc/_chapters/binder_msgs.adoc

```diff
@@ -2919,3 +2919,15 @@ Use of predefined UDF <name> is deprecated and this function will be removed in
 
 *Recovery:* See message.
 
+[[SQL-4997]]
+== SQL 4997
+
+```
+The <number> operand of builtin-func <name> must be great than <intger value>.
+```
+*Cause:* You attempted to use the random function, which some operand of this has range.
```

Comment:
The mention of the "random" function looks like a cut-and-paste error. Suggested wordsmith: "You attempted to use an invalid value for the given operand of the given function."

Reply:
Thanks, DaveBirdall, i will commit and pull again after fixed this issues.